.travis.yml

```diff
@@ -60,22 +70,28 @@ matrix:
     - python -m pip install -r cirq/contrib/contrib-requirements.txt
     - python -m pip install -r dev_tools/conf/pip-list-dev-tools.txt
     script: check/pytest --benchmark-skip --actually-quiet
-  - os: linux
+  - stage: test
+    os: linux
     env: NAME=doctest
     python: '3.6'
     install:
     - pip install -r requirements.txt
     - pip install -r cirq/contrib/contrib-requirements.txt
     script: check/doctest -q
+  - stage: deploy
+    os: linux
+    env: NAME=dev-release
+    python: '3.7'
```

Comment:
why 3.7?  I think we say we support python 3.6

Reply:
I just picked one.  This would just be the python version that runs setup.py.

Reply:
ah k